### ISSUE TITLE
Display Base Name Request Details

### ISSUE THREAD

**Issue by JohnamLane** [bcgov/name-examination#117]
[Information About NRO Applicant and other contacts.docx](https://github.com/bcgov/name-examination/files/1872899/Information.About.NRO.Applicant.and.other.contacts.docx)

#### **Story**
As an examiner I want to be able to see the name request submission details so that I have pertinent information at  my finger tips while I am examining and making a decision.
##### BDD Pseudo-Code (GIVEN "context" ... AND "more context" ... WHEN "this action occurs" ... THEN ..."result")
GIVEN the NR is the current one and an authenticated examiner is working on and it has a status "IN PROGRESS" 
WHEN the Name Request Details are Retrieved 
THEN display the NR fields

##### _**Acceptance Criteria**_
Display the following fields. If field is empty, simply show the label.

Display the following fields related to the NR:
- NR Number VARCHAR2(10, BYTES)
- All 3 name choices (-select the first name choice, show as highlighted) VARCHAR2(150) -each name
- Request Type (e.g. Corporation - Incorporation/Amalgamation, see NAMESDB.REQUEST_TYPE) 
  -Request Type CD -VARCHAR2(10) - Julia can decide whether to use ling or short desc.
 - Request Type SHORT_DESC VARCHAR2(60)
-  Request Type FULL_DESC VARCHAR2(100)
- Applicant Information: Applicant=PARTY_TYPE_CD='APP'
      -Applicant Last Name -VARCHAR2(50, BYTE)
    - Applicant FIRST_NAME VARCHAR2(50, BYTE)
   - Applicant MIDDLE_NAME VARCHAR2(50, BYTE)
   - Applicant PHONE_NUMBER	VARCHAR2(30 BYTE)
   - Applicant FAX_NUMBER	VARCHAR2(30 BYTE)
   - Applicant EMAIL_ADDRESS	VARCHAR2(75 BYTE)
 -  Applicant ADDRESS_ID	NUMBER(10,0)  (See ADDRESS sample in the attachment above to get columns)
  - CONTACT	VARCHAR2(150 BYTE)
 -  CLIENT_FIRST_NAME	VARCHAR2(50 BYTE)
- CLIENT_LAST_NAME	VARCHAR2(50 BYTE)
- XPRO_Jurisdiction VARCHAR2(40,BYTE) No look-up table full text eg. ONTARIO
- Nature of Business - NATURE_BUSINESS_INFO VARCHAR2(1000,BYTE)
- Additional Information ADDITIONAL_INFO VARCHAR2(150,BYTE)
- NUANS Name requested (Name Choice Field - in this case there will only be one name choice and it must be the name used in the home jurisdiction)
For Alberta and SK (NWPTA registration as well)
 - NUANS_NUM VARCHAR2(20, BYTE)
- NUANS_EXPIRATION_DATE DATE
- ASSUMED_NUANS_NUM	VARCHAR2(20 BYTE)
- ASSUMED_NUANS_NAME	VARCHAR2(255 BYTE)
- ASSUMED_NUANS_EXPIRATION_DATE	DATE

- If the PARTNER_NAME_SYSTEM.PARTNER_NAME_TYPE_CD = 'AS', then show a check box labelled:
"An assumed numbered name will be used in Alberta."
Substitute Saskatchewan if the name is for SK.

Allows the applicant to register in SK and Alberta at the same time form NRO which means there mist be two rows for these fields. TILMA_IND='Y' for these as well but not required on the front end

-Request Status REQUEST_STATE.STATE_TYPE_CD VARCHAR2(10,BYTES) 
    Valid values: D	Draft	Draft
                        C	Cancelled	Cancelled
                        H	Held	Under examination - not processed.
                       COMPLETED	Completed	Completed - accepted or rejected.
                       E	Expired.	Request name expired before being consumed.
                      HISTORICAL	Historical	Historical
                      IN PROGRESS (New - we are dding this one )

- Name Choice Status NAME_STATE.NAME_STATE_TYPE_CD VARCHQAR2(15,BYTES)
Current Values are:
A	Approved	Examined, approved
C	Cond. Appr.	Examined - Conditionally Approved
R	Rejected	Examined and rejected
NE	Not Examined	Not examined - in the queue or another choice approved

- Examiner Assigned REQUEST_STATE.EXAMINER_IDIR VARCHAR2(8,BYTES)
- Priority (Regular, Priority, JumpQ)  - we may want to call them something different but that is what they are now.
  - PRIORITY_CD VARCHAR2(2) - No Look-up Table in namesp.
- values can be: PQ, RJ, RQ, PJ  I think these mean Priority Queue, Regular Jump, Regular Queue, Priority Jump.  These are current in NRO.
 
- SUBMIT_COUNT NUMBER(3,0) ( used to manage re-submissions
- PREVIOUS_REQUEST_ID NUMBER(10,0) (Linked/Previous NR for managing re-submissions and maintaining associations)
-reservation_count (new field) NUMBER(3,0) (set when approved)
-EXPIRATION_DATE DATE (can be NULL) (will be read-only unless it is an N status=Approved and consumption_date=NULL)

Definition of Done
 - [x] Analysis/Design
- [ ] DB
- [ ] UI
 - [ ] Integration Points
 - [ ] Logic
 - [ ] Test Script

##### _**Issue Checklist**_
- [x] BDD Pseudo-Code 
- [x] Estimate 
- [x] Priority Label 
- [x] Story Label
- [x] Linked EPIC:

**Comment by LJTrent** [comment]
@juliainkster @rarmitag @thorwolpert 

Hi,
I have updated the field list in the story so we all understand all of the fields that need to be included in the NR details UI.  I have listed them as they come from namesp/NRO.  As we all know the NRO data model leaves a little to be desired so I assume that Thor will be implementing a different data model/set of tables in Postgres.  Richard will need to use Thor's data model however this does give you all of the fields that need to be addressed for NR details.  There is a data model for namesp including the mainframe tables on the window outside of the room that might help with these tasks.
